custom/icds_reports/tests/agg_tests/outputs/agg_awc_daily_2017-05-28_sorted.csv

```diff
@@ -50,15 +50,15 @@ st2,d2,b3,s6,a38,5,2017-05-28,211,62,63,,,,4,4,2,2,5,1,5,1,0,1,1,1,1,1,1,,1,6,0,
 st2,d2,b3,s6,a46,5,2017-05-28,162,9,9,,66,66,7,7,3,3,0,0,0,0,0,1,1,1,1,1,1,,0,76,0,0,0,0,0,
 st2,d2,b3,s6,a6,5,2017-05-28,,21,21,,53,53,10,10,7,7,0,0,0,0,0,1,0,0,0,0,0,,9,70,0,0,0,0,0,
 st2,d2,b3,s6,All,4,2017-05-28,515,179,180,,194,194,27,27,18,18,8,4,8,4,1,6,1,1,1,1,4,,23,239,0,0,0,0,0,12
-st2,d3,All,All,All,2,2017-05-28,610,261,263,,358,358,47,47,39,39,8,2,8,2,0,12,1,1,1,2,5,,103,444,0,0,0,0,0,
-st2,d3,b4,All,All,3,2017-05-28,610,261,263,,358,358,47,47,39,39,8,2,8,2,0,12,1,1,1,2,5,,103,444,0,0,0,0,0,
+st2,d3,All,All,All,2,2017-05-28,610,261,263,,359,359,47,47,39,39,8,2,8,2,0,12,1,1,1,2,5,,103,445,0,0,0,0,0,
+st2,d3,b4,All,All,3,2017-05-28,610,261,263,,359,359,47,47,39,39,8,2,8,2,0,12,1,1,1,2,5,,103,445,0,0,0,0,0,
 st2,d3,b4,s7,a15,5,2017-05-28,,19,19,,19,19,2,2,0,0,1,0,1,0,0,1,0,0,0,0,0,,1,21,0,0,0,0,0,
 st2,d3,b4,s7,a23,5,2017-05-28,,6,6,,49,49,6,6,4,4,0,0,0,0,0,1,0,0,0,0,0,,15,59,0,0,0,0,0,
 st2,d3,b4,s7,a31,5,2017-05-28,,21,21,,25,25,3,3,6,6,1,1,1,1,0,1,0,0,0,0,0,,22,34,0,0,0,0,0,
 st2,d3,b4,s7,a39,5,2017-05-28,146,17,17,,36,36,4,4,3,3,2,1,2,1,0,1,1,1,1,1,1,,9,43,0,0,0,0,0,
 st2,d3,b4,s7,a47,5,2017-05-28,144,50,50,,11,11,3,3,3,3,3,0,3,0,0,1,1,1,1,1,1,,2,17,0,0,0,0,0,
-st2,d3,b4,s7,a7,5,2017-05-28,,13,14,,56,56,5,5,1,1,0,0,0,0,0,1,0,0,0,0,0,,8,62,0,0,0,0,0,
-st2,d3,b4,s7,All,4,2017-05-28,290,126,127,,196,196,23,23,17,17,7,2,7,2,0,6,1,1,1,1,2,,57,236,0,0,0,0,0,
+st2,d3,b4,s7,a7,5,2017-05-28,,13,14,,57,57,5,5,1,1,0,0,0,0,0,1,0,0,0,0,0,,8,63,0,0,0,0,0,
+st2,d3,b4,s7,All,4,2017-05-28,290,126,127,,197,197,23,23,17,17,7,2,7,2,0,6,1,1,1,1,2,,57,237,0,0,0,0,0,
```

Comment:
As offlined After adding this fixture these changes should be reverted back. which they are not. Which is because of a bug in the migration table agg you wrote in last PR.
[here](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/aggregation_helpers/distributed/migration_forms.py#L18) while calculating the next month, it is making it jan of that year therefore the table is endded up created empty. Could you please fix it.

Reply:
And can you make that fix in a separate Pr, so that we merge that PR and start the data filling.